addons/mail/static/src/messaging/component/discuss/discuss_tests.js

```diff
@@ -5298,11 +5265,7 @@ QUnit.test('moderation: sent message accepted in moderated channel', async funct
         res_id: 20,
     }];
 
-    await this.start({
-        session: {
-            partner_id: 13,
-        },
-    });
+    await this.start();
```

Comment:
Again, would provide session data.
I think logic doesn't care of it, but this test simulates a message of itself being accepted, so message author partner id should match current partner id.

Reply:
The test works even if it is not a message of the current user.

As long as it is a "pending message that is known" being accepted, the treatment should be the same.

It appears that in our business code, other users typically don't see pending messages, but there is no reason to test a particular case if the general case works as here.

Note that our `o-author` modifier name might be misleading, it actually means "non-moderator" (as per the condition to display it).